src/components/Player/Pages/Histograms/Histograms.jsx

```diff
@@ -19,14 +19,28 @@ const getMedian = (columns, midpoint) => {
   return medianCol && medianCol.x;
 };
 
+const getMax = (columns) => {
+  let max = 0;
+  let maxColumn;
+  columns.forEach((col) => {
+    max = Math.max(max, col.games);
+    if (max === col.games) {
+      maxColumn = col.x;
+    }
+  });
+  return maxColumn;
+};
+
 const getSubtitleStats = (columns) => {
   const total = columns.reduce((sum, col) => (sum + col.games), 0);
-  const median = getMedian(columns, total / 2);
-  let str = `${strings.heading_total_matches}: ${total}`;
-  if (median !== undefined) {
-    str += `, ${strings.heading_median}: ${median}`;
-  }
-  return str;
+  const stats = [
+    [strings.heading_total_matches, total],
```

Comment:
I suggest using objects rather than arrays here so indexing into them is more readable and doesn't break if things are reordered.